core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -6274,6 +6274,63 @@ private void checkNullOperand(SqlTester tester, String op) {
     tester.checkNull("substring(cast(null as varchar(1)),1,2)");
   }
 
+  @Test public void testRegexpSplitToArrayFunction() {
+    final SqlTester t = tester(SqlLibrary.POSTGRESQL);
+    t.setFor(SqlLibraryOperators.REGEXP_SPLIT_TO_ARRAY);
+    t.checkString(
+        "regexp_split_to_array('','a')",
+        "[]",
+        "VARCHAR(0) NOT NULL ARRAY NOT NULL");
+    t.checkString(
+        "regexp_split_to_array('badac','a')",
+        "[b, d, c]",
+        "VARCHAR(5) NOT NULL ARRAY NOT NULL");
+    t.checkString(
+        "regexp_split_to_array('badac','ad')",
+        "[b, ac]",
+        "VARCHAR(5) NOT NULL ARRAY NOT NULL");
```

Comment:
What about one of the arguments is `NULL`? Could you please add a test for it?

Reply:
I missed that case, thanks @chunweilei .
PR updated.